.github/workflows/openliberty-ci.yml

```diff
@@ -20,14 +20,14 @@ on:
 env:
   LANG: en_US.UTF-8
 jobs:
-  build:
-    name: Build Open Liberty
-    runs-on: ubuntu-latest
-    outputs:
+  setup: #Validates copyright headers, selects test-os, selects java version, and sets up test categories. 
```

Comment:
I'd rather leave this stuff in the core `Build` job because there is such a small amount of work occurring in `Setup`. Using public runners this completes in about 70 seconds it looks like, but most of that time is spent provisioning the new machine and cloning the repo, When we switch to self-hosted runners this will likely become more of an issue where provisioning new jobs are slightly slower.